cdc/owner/changefeed.go

```diff
@@ -306,48 +336,48 @@ func (c *changefeed) checkStaleCheckpointTs(ctx cdcContext.Context, checkpointTs
 	return nil
 }
 
-func (c *changefeed) tick(ctx cdcContext.Context, captures map[model.CaptureID]*model.CaptureInfo) error {
-	adminJobPending := c.feedStateManager.Tick(c.state, c.resolvedTs)
-	preCheckpointTs := c.state.Info.GetCheckpointTs(c.state.Status)
+func (c *changefeed) tick(ctx cdcContext.Context,
+	cfInfo *model.ChangeFeedInfo,
```

Comment:
Seems that `cfStatus` and `cfInfo` are already updated into the changefeed sturct in the `Tick` method. So there is no need to pass them into `tick` method explicitly.

Reply:
done